Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -192,7 +190,7 @@ public async Task<IActionResult> Update(int id, [FromBody]UpdateKeyViewModel mod
         }
 
         [HttpPost("{id}")]
-        [ProducesResponseType(typeof(Key), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(KeyInfo), StatusCodes.Status200OK)]
         public async Task<IActionResult> Delete(int id)
```

Comment:
returns null?

Reply:
Thinking about it now since this endpoint in particular returns `null` I think we should not have a ResponseType for it?

Reply:
@sprucely What do you think? ☝️

Reply:
Oops, didn't see this last week. There's an overload of ProducesResponseType that takes just the status code. Should probably at least use that.